app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppState.kt

```diff
@@ -78,6 +84,7 @@ fun rememberNiaAppState(
 @Stable
 class NiaAppState(
     val navController: NavHostController,
+    val niaNavigator: NiaNavigator,
```

Comment:
Using existing `NiaAppState` to hold and provide the navigator holder. `appState` is already passed around so makes access to the navigator easy in this setup.